test/passing/assign_comment-begin_line.ml.ref

```diff
@@ -0,0 +1,35 @@
+(* see https://github.com/ocaml-ppx/ocamlformat/issues/1088 *)
+
+let _ =
+  r (* _________________________________________________________________ *)
+    := 1
+
+let _ =
+  r
+    (* _________________________________________________________________ *)
+    (* _________________________________________________________________ *)
+    := 1
+
+let _ =
+  r
+    := (* _________________________________________________________________ *)
```

Comment:
This comment exceed the margin by 1. Breaking the line won't help so maybe it's ok ?

Reply:
Yes there is already enough breaks before and after the comment, at some point the user needs to put spaces in comments :D